_posts/2016-03-29-introducing-outline-audit.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Introducing Outline Audit"
+author: hugo-giraudel
+layout: post
+---
+
+We recently had some minor issues in a project where the final document outline on some pages was somewhat broken. I investigated a little, and ended up building [a tiny tool](https://github.com/edenspiekermann/outline-audit) to audit the document outline. More on that in a bit, but first…
+
+## What’s the document outline?
+
+The document outline is the theoretical schema constructed from the structure of a document (such as a web page), mostly based on headings. It matters because assistive technologies like screen readers heavily rely on this to give context and help navigation. It also matters for search engines, so they can locate and index meaningful content first.
```

Comment:
"screen readers rely heavily on this" reads a bit better (they're both right though)